package.json

```diff
@@ -147,7 +148,7 @@
   },
   "scripts": {
     "test": "./scripts/test.sh",
-    "test-unit": "nyc --reporter=text -x **/assets/** -x **/test/** ./node_modules/mocha/bin/mocha \"test/unit/**/**.test.js\"",
+    "test-unit": "nyc --reporter=text -x **/assets/** -x **/test/** ./node_modules/mocha/bin/mocha --timeout 90000 \"test/unit/**/**.test.js\"",
```

Comment:
Do we need this change ?

Reply:
@barshan23 Yes, as stripe definition is taking ~50 sec of time to validate, we'll need to increase this.